lib/bolt_server/plan_app.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
```

Comment:
This probably isn't where we want it to live if it's going to be called the Plan Executor. Would that be `lib/plan_executor/app.rb`?